### PR TITLE
Deploy Mintmaker to production

### PR DESCRIPTION
**Pull request by qixiang**

### CODE REVIEW

**Comment by qixiang** [comment]
This will remain in draft state until the staging deployment is verified.

File: argo-cd-apps/base/member/infra-deployments/mintmaker/mintmaker.yaml [hunk 1]
```diff
@@ -17,6 +17,10 @@ spec:
               elements:
                 - nameNormalized: stone-stage-p01
                   values.clusterDir: stone-stage-p01
+                - nameNormalized: stone-prod-p01
+                  values.clusterDir: stone-prod-p01
```
Comment (qixiang) [inline argo-cd-apps/base/member/infra-deployments/mintmaker/mintmaker.yaml:21]: Seems p01 will be decommissioned, should we remove this?

Comment (scoheb) [reply argo-cd-apps/base/member/infra-deployments/mintmaker/mintmaker.yaml:21]: let it there for now. when decommissioned, someone will clean this up.

**Comment by scoheb** [comment]
this file is missing: https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build-service/production/base/pipelines-as-code-secret-patch.yaml in mintmaker

**Comment by qixiang** [comment]
> this file is missing: https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build-service/production/base/pipelines-as-code-secret-patch.yaml in mintmaker

Good catch, updated.

**Review by hugares — APPROVED** [review]
/lgtm

**Review by mmorhun — APPROVED** [review]
/lgtm

**Pull request merged**